puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/packaging/pulpcore-packaging-rpm-nightly.yaml

```diff
@@ -0,0 +1,16 @@
+---
+- job:
+    name: pulpcore-packaging-rpm-nightly
+    project-type: pipeline
+    concurrent: true
```

Comment:
Does it make sense to run these concurrently? Won't you end up with duplicate builds?

Reply:
the pipeline file has concurrent no, and that seems to win anyways :see_no_evil: